floris/simulation/turbine.py

```diff
@@ -472,9 +472,9 @@ class PowerThrustTable(FromDictMixin):
         ValueError: Raised if the power, thrust, and wind_speed are not all 1-d array-like shapes.
         ValueError: Raised if power, thrust, and wind_speed don't have the same number of values.
     """
-    power: NDArrayFloat = field(default=[], converter=floris_array_converter)
-    thrust: NDArrayFloat = field(default=[], converter=floris_array_converter)
-    wind_speed: NDArrayFloat = field(default=[], converter=floris_array_converter)
+    power: NDArrayFloat = field(factory=list, converter=floris_array_converter)
```

Comment:
I guess I'd say we should have a unified approach now that I see it's a separate approach for the same results in each of the files so far (love Python for this!), at least to make it easier for future developers.